### ISSUE TITLE
New links design

### ISSUE THREAD

**Issue by andersonjeccel** [mautic/mautic#13153]
### Description
The current links design is outdated and does not meet modern usability standards; we aim to create a fresh, more visually appealing, and user-friendly links design.

User feedback:

> More intuitive (links in small, grey typeface that you don’t know are links is one example)

### Impact
Implementing the new links design will not only improve the aesthetics of the interface but also enhance navigation and user interaction, resulting in a more engaging and efficient user experience.

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by RCheesley** [comment]
Still relevant

**Comment by andersonjeccel** [comment]
Done in #13770
